### PR TITLE
Deep Lab V3 Guide

### PR DESCRIPTION
**Pull request by cyber-aman**
\<insert pull request description here\>

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [ ] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports?
- [ ] All `GIS` object instantiations are one of the following?
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [ ] Code refactored & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

File: guide/14-deep-learning/how_deeplabv3_works.ipynb [hunk 1]
```diff
@@ -0,0 +1,230 @@
+{
```
Comment (priyankatuteja) [inline guide/14-deep-learning/how_deeplabv3_works.ipynb]: <blockquote>INTRODUCTION</blockquote><p>Replace it with How DeeplabV3 Works ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (priyankatuteja) [inline guide/14-deep-learning/how_deeplabv3_works.ipynb]: <blockquote>&nbsp;guide "How U-net Works" where an intuition of Semantic Segmentation explained.</blockquote><p>looks like an incomplete statement</p><p> </p><blockquote>To understand contents of this guide, we assume that you have some basic understanding of the convolutional neural networks (CNN) concepts. You can refresh your CNN knowledge by going through this short paper “A guide to convolution arithmetic for deep learning”.</blockquote><p>Should go under Note section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (priyankatuteja) [inline guide/14-deep-learning/how_deeplabv3_works.ipynb]: <blockquote>Atrous Convolution was introduced in DeepLab as a tool to adjust/control effective field-of-view of the convolution. It uses a parameter called ‘atrous/dilation rate’ that adjusts field-of-view. </blockquote><p>Atrous Convolution is introduced in DeepLab as a tool to adjust/control effective field-of-view of the convolution. It uses a parameter called ‘atrous/dilation rate’ that adjusts field-of-view. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (priyankatuteja) [inline guide/14-deep-learning/how_deeplabv3_works.ipynb]: <p>research paper citations should follow APA style </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by priyankatuteja** [comment]
@cyber-aman Looks good! I have suggested some minor changes.

File: guide/14-deep-learning/how_deeplabv3_works.ipynb (hunk 1 above)
Comment (cyber-aman) [reply guide/14-deep-learning/how_deeplabv3_works.ipynb]: <p>Done</p>

Comment (cyber-aman) [reply guide/14-deep-learning/how_deeplabv3_works.ipynb]: <p>Done</p>

**Pull request merged**